### DISCUSSION TITLE
french stream

### DISCUSSION THREAD

**Discussion by jvvince** [aclap-dev/video-downloadhelper#1512] [Help and Q&A]
### Your question

Hi, for any reasons DownloadHelper doesn't work anymore with https://vvw.french-stream.bio/films/ 
PS: i use Firefox Hoxx VPN to connect to this site, few weeks ago it was working.
What's wrong ? Thanks for help.

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: fr
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"dffbaeb034fb0c0b"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "D:\\V I D E O S",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "D:\\V I D E O S"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": false,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": true
    },
    "use_sidebar": true,
    "open_count_store": 7,
    "successfull_dl": 59
}

**Comment by Mattmjs** [comment]
What do you mean by doesn't work anymore ? And give a specific example, the link you've given is a just page with films on it.

**Reply by jvvince** [reply]
Hello, few weeks ago when i was playing a movie on this site it was possible to download the movie with DownloadHelper but now when i try to download i have an error (see attached pic). I have the same problem with all movies.
As example here is a link : https://vvw.french-stream.bio/15027-chute-libre-film-streaming-complet-vf.html

![0](https://github.com/user-attachments/assets/20f7480b-206d-4557-a545-4019d90294e2)

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 24H2 build 26100.2454, Firefox 133.0 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.  I'm using neither a VPN nor a proxy.  I believe I've seen reports in here that proxies can be a problem.  But I believe I've seen people posting in here that VDH works fine with VPNs.

Here's what I'm looking at right now:

![#01](https://github.com/user-attachments/assets/79adc29b-5c0a-4118-afae-b23882449e3d)

That is a horrifically slow download.  But at least it's downloading.  I'm guessing it's going to take at least the duration of the film, which is shown in that image as 1:52:31, to complete.  If I don't post again, assume it worked & played fine in VLC.  I am not planning to sit & watch the movie.  I will sample it at intervals to make sure it has both video & audio at each sample point.  I will also play the very end of the video to make sure I've got it all.  I'm expecting to see closing credits, a copyright notice, & a fade to black.

If I do not see all of that, I'll post again.  Assume that no news will be good news.

**Reply by GrampaWildWilly** [reply]
I just noticed your Firefox is badly out of date.  You really ought to be using the latest release.

You are using CoApp 2.0.19.  That should not be a problem.  I would not update that.  I have 2.0.20 because I found it here on GitHub.  But after I installed it, Paul told me it's "not ready for prime time."  I don't know what that means, but I took that as a warning to be prepared to drop back to 2.0.19 if any problems arise.  I haven't had any problems so I don't know what Paul was worried about.  But I would not recommend anybody update to 2.0.20.  2.0.20 is probably not the source of either problems or solutions.

Since you are on Firefox, I would recommend you try the VDH beta.  That's what I have & I appear to be successfully downloading your movie.

**Reply by jvvince** [reply]
Hi, 
It is out of date because I think there are no more updates for Firefox with Windows7 so I thought there was a problem because I was operating on an old Window7 PC.
I installed the latest versions of DLH and coApp + licence on a Windows10 machine and I still have the same problem. If i switch to old DLH interface in fact I see that the media is not detected. By the way, I don't know how to get back to the new DLH interface.
DLH works with Youtube video so i wonder if the problem is due to the Hoxx VPN in Firefox ??? 

Well, the problem stays here ?! thanks in advance for help.

**Reply by GrampaWildWilly** [reply]
Ah.  Windows 7.  That detail is not displayed by the `Copy information to clipboard` data, just that you're running Windows.  I wish that function gave more detailed information for the operating system.  It's not specific for Mac or Linux either.

No, VDH should work on W7.  But there is a special version of the CoApp for W7.  I assume that's the one you have.  You can find it in the Wiki.  There's a link to our Wiki at the top of every page of this forum.

If you have selected the `Use legacy UI` setting, then switching back to the new VDH interface is done as I describe over here: #1317.

I still recommend trying the VDH beta.  You can find that in the Wiki as well.

The only way to find out if the VPN is interfering with your downloads is to turn the VPN off & try again.  One user recently posted #1477 in which he described his experience with a VPN.  Perhaps that might help you.

**Reply by Mattmjs** [reply]
Windows NT 6.1 indicates it's windows 7, windows 10 and 11 are both NT 10.0, there is a Wikipedia page about it.
I think the problem is the browser vpn, turning it off or using a proper desktop vpn app would be the way to go.
This may be because browser vpn extensions encrypt your web browser activity while the rest of the computer is unencrypted,
perhaps this doesn't allow the extension to properly interact with companion app.

**Reply by Mattmjs** [reply]
The other thing is both of you in your screenshots used the source called DOOD, this is heavily throttled that the download
could terminate early. Use the one called Voe.

**Reply by jvvince** [reply]
Grampa thank you for your engagement and for the links to articles, i'll read them, just one thing : have the same problem on an other computer with Windows10.
Mattmjs i really think the problem comes with VPN because DLH works on Youtube (no VPN) but on french-stream site i can't access without VPN.
i have the same problem with DOOD, VOE, UQLOAD etc....
What is curious is that some time ago it worked,
How can I downgrade the version of DLH without resetting the license, I lost 2 tries trying.
Thank you both for help

**Reply by GrampaWildWilly** [reply]
Yeah, DOOD was really slow.  But the VDH download I ran did complete.  It took about an hour & a quarter but it did complete.  Being slow by itself should not cause a download, not in VDH nor in any other tool, to terminate early.  Other factors might do that, like if your ISP imposes a time limit on your downloads.  Also, long downloads just increase the probability that a line error will occur.  I suppose I was lucky that didn't happen to me.  The speed I was getting with this thing was worse than the bandwidth I used to get on my old DSL connection.  That one did regularly suffer line errors.  I look back on those bad old days & can't believe I put up with that for as long as I did.

Strange but I was able to get to the site without a VPN.  I gave up on VPNs several years ago.  Despite having the VPN, the web sites I wanted to access still told me their content was not available in my country.  VPNs maintain points of presence in the various countries.  Webmasters know the IP addresses of those points of presence & treat them like foreign IP addresses, which means you get geoblocked exactly the same with the VPN as without.  It was quite disappointing.

You should keep your VDH license in a safe place where you can get it whenever you want.  By get it I mean copy/paste it.  That's all you have to do when you install a different version of VDH, upgrade or downgrade.  Actually, I'm not entirely sure you will need to re-enter your VDH license.  As long as you don't outright remove your current version of VDH first, I think there's a chance you'll just still be registered.  But I could be wrong.

**Comment by Mattmjs** [comment]
Before you downgrade, you should try this first. Disable the vpn in Firefox then clear history & cache.
In Firefox settings look for DNS over HTTPS in the privacy & security section. Enable Increased Protection option.
Does it work after this ?

**Reply by jvvince** [reply]
Hi Mattmjs, thousand thanks ! it works.
Does this setting change have an impact on other things ?
A big thank to you Grampa too to have took on your time to to help me.
Have a great day both.

**Reply by Mattmjs** [reply]
It shouldn't have any effect, all it does is changing the DNS from the isp to another dns provider if you have trouble
resolving a site.

**Reply by jvvince** [reply]
Thank you.
